cron/k8s/worker.yaml

```diff
@@ -47,3 +47,8 @@ spec:
           limits:
             cpu: 12
             memory: 12Gi
+  strategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
If there is only one replica this will not work. We need at least 2 to rolling update to work.

Reply:
Doesn't this imply a `n+1` RollingUpdate strategy here? Basically, what I want to have here is a strategy that brings up a new instance (`maxSurge` = 1) and kills an older one (`maxUnavailable` = 1).

Let me know if this isn't doing what I expect it to do and what change I need to make here for it work that way.

Reply:
Have you tried this? I am not aware of the max surge with this option. As long as you have tried it and it works I am fine.

Reply:
I tried it through the UI and it worked. But then again, we only have 1 worker instance running. 

Maybe we keep this as is for now? My main aim is unblock auto deployment of new binaries when they become available. If the behavior breaks with >1 worker instances, we can revisit then. Wdyt?

Reply:
Sounds good.